webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -280,4 +278,28 @@ test.describe('Experiment List', () => {
     // });
     // await test.step('Hyperparameter Search', async () => {});
   });
+
+  test('DataGrid Action Pause', async () => {
+    // create paused experiment
+    detExecSync(
+      `experiment create ${fullPath('examples/tutorials/mnist_pytorch/adaptive.yaml')}  --paused`,
+    );
+    await waitTableStable();
```

Comment:
i think we should be waiting for a row to be present here, right? it's more like we're waiting for the table to refresh.
you might want to try `getRowByColumnValue` to make sure the experiment is present, but i can see you're getting the first experiment here. experiments should be sorted newest on top, so it should be fine.

Reply:
can you make sure the number of experiments present is +1 after this step?